integration/main_test.go

```diff
@@ -396,3 +401,39 @@ func waitUntilNotFound(ctx context.Context, obj client.Object) error {
 		return fmt.Errorf("resource is still running: %s", obj.GetName())
 	}, retry.Context(ctx), retry.Attempts(0), retry.MaxDelay(2*time.Second))
 }
+
+func execCommand(ctx context.Context, containerName string, cmd []string) (string, error) {
```

Comment:
Pass pod and namespace name here. The namespace can be derived from `testNamespace(t)` by the caller, pod name is static.